src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,77 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as currentTodoActions } from '../../features/currentTodo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
 export const TodoModal: React.FC = () => {
-  return (
-    <div className="modal is-active" data-cy="modal">
-      <div className="modal-background" />
+  const [hasLoadingError, setLoadingError] = useState(false);
+  const [currentUser, setCurrentUser] = useState<User | null>(null);
+  const currentTodo = useAppSelector(state => state.currentTodo);
```

Comment:
It's better if you get this value as a prop of the component just like you did in `<TodoList />`.